src/public/icons/terminal.svg

```diff
@@ -0,0 +1,8 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I think icon usually don't have width and height defined in our codebase, worth double check
i know there is an option in svgomg to remove them